tests/playbooks/tests_switch_provider.yml

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# This file was generated by ensure_provider_tests.py
+---
+# set network provider and gather facts
+- hosts: all
+  name: Switch initscripts provider to nm
+  vars:
+    interface: LSR-TST-br34
+  tasks:
+    - name: set fact to use initscripts network_provider
+      set_fact:
+        network_provider: initscripts
+      tags:
+        - always
+    - name: "Create test bridge {{ interface }} via initscripts provider"
+      include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: "{{ interface }}"
+            state: up
+            type: bridge
+            ip:
+              dhcp4: false
+              auto6: false
+    - include_tasks: tasks/assert_device_present.yml
+    - name: "Take the profile {{ interface }} down via initscripts provider"
+      include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: "{{ interface }}"
+            persistent_state: absent
+            state: down
+            type: bridge
```

Comment:
This is the duplication of the content of `down_profile.yml`. Since the tests need to be cleaned up anyhow, it might be fine to add more tech debt here. IMHO, it would be better to move more towards a cleaner test format like https://github.com/linux-system-roles/network/blob/main/tests/playbooks/tests_states.yml or something better than that.
By checking that file I noticed, that it contains an exception for the case that this PR is supposed to fix, so that should be removed, too:
https://github.com/linux-system-roles/network/blob/main/tests/playbooks/tests_states.yml#L129

Reply:
I have done several changes. down_profile keeps doing the same thing than before. down_profile+delete_interface is a new file that is importing down_profile and using delete_interface task. This is avoiding duplicate code.

Reply:
Also removed the exception in `tests_states.yml`